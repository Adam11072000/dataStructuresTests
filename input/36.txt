init 28
deleteImage -8
deleteImage 42
deleteImage 114
quit