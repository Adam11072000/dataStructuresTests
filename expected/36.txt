init done.
deleteImage: INVALID_INPUT
deleteImage: FAILURE
deleteImage: FAILURE
quit done.
